src/main/java/com/exactpro/th2/check1/Check1Handler.java

```diff
@@ -174,4 +185,64 @@ public void submitNoMessageCheck(NoMessageCheckRequest request, StreamObserver<N
             responseObserver.onCompleted();
         }
     }
+
+    @Override
+    public void submitMultipleRules(MultiRulesRequest request, StreamObserver<MultiRulesResponse> responseObserver) {
+        if (logger.isInfoEnabled()) {
+            logger.info("Rules list for request '{}' started", MessageUtils.toJson(request));
+        }
+
+        final boolean isResponseNeeded = !request.getOmitResponse();
+        final MultiRulesResponse.Builder responseBuilder = MultiRulesResponse.newBuilder();
+
+        if (!isResponseNeeded) {
+            responseObserver.onNext(responseBuilder.build());
+            responseObserver.onCompleted();
+        }
```

Comment:
Honestly, this looks a bit strange... I would think that having two methods (one returns MultiRulesResponse and another one returns Empty in response) would be better in terms of clarity for users

Reply:
added `postMultipleRules` method